golang/defaults.yaml

```diff
@@ -1,7 +1,7 @@
 golang:
   prefix: /usr/local
   go_root: /usr/local/go
-  go_path: /usr/local/go/bin
+  go_path: /usr/local/golang/packages
```

Comment:
From my limited research, `go_path` is something that will always be configured via pillar data because its a personal choice.  I considered`/usr/local/golang/dir` a better default value than  `/var/lib/golang`.  And `~/golang` is maybe even better so I added to pillar.example.

No, actually go_path is like "here is where I put my src code" because I am GO developer.  If you are not GO developer you probably don't care about go_path.

Reply:
Ummm, no, it's used to [build a `shell profile`](https://github.com/saltstack-formulas/golang-formula/blob/master/golang/init.sls#L84-L94) that is installed globally in the system, and that var [is used to add the golang's binary path to the system's `$PATH`](https://github.com/saltstack-formulas/golang-formula/blob/master/golang/files/go_profile.sh#L6-L7)